integr8ly/cloud-resource-operator
# Update redis openshift deployment memory request

## Overview

Currently intly installs on PDS fail due exceeding container memory limits. This reduces the default we set

## Verification

- Clone this branch
- Run `make cluster/prepare`
- Run `make cluster/seed/workshop/redis`
- Run `make run`
- Ensure the redis deployment has a memory request of 6gi
